product_docs/docs/eprs/7/01_introduction/04_supported_jdk_versions.mdx

```diff
@@ -11,7 +11,7 @@ Replication Server is certified to work with the following Java platforms:
 | SLES 12                                 | Red Hat OpenJDK 8                                                                                               |
 | PPCLE RHEL 8                            | Red Hat OpenJDK 8 |
 | RHEL 7 and 8                            |  -   Red Hat OpenJDK 8     <br /> <br /> -   Oracle JDK 8     <br />         |
-| Windows 2012 R2, 2016, and 2019         | Red Hat OpenJDK 8                                                                                          |
+| Windows 2016, 2019                            | Red Hat OpenJDK 8                                                                                          |
```

Comment:
Based on a discussion with @kelpoole, when normal support for a platform ends, even if extended support for it is still available, EDB's normal policy is to discontinue support for our software on that platform.  As such, this change should be reverted back to the previous version where only Windows 2019 listed is listed.

In addition the "Windows 2019" listing should be changed to "Windows Server 2019".